deployments/docker/README.md

```diff
@@ -69,11 +124,12 @@ The deployment may be configured by changing the settings inside
 [deployment.cfg](./deployment.cfg). See the comments inside that file for the
 explanation of the individual configuration values.
 
-> **Note**: The individual services rely on configuration inside
-> `config.yaml.template`. This currently uses the `sqlite3` driver for the
-> store backend, which limits request throughput and makes the deployment
-> unsuitable for production or performance testing (for the latter, one can
-> modify the template to use `memory` backend for the stores).
+> [!NOTE]
+> The individual services rely on configuration inside `config.yaml.template`.
+> This currently uses the `sqlite3` driver for the store backend, which limits
```

Comment:
Should we not have the options for other drivers at least in the documentation

Reply:
Not for the Docker deployment, as Postgres/MariaDB do not currently run in the deployment. In the future, we may wish to switch the deployment to use one of those, rather than `sqlite3`; but that's a bigger change.